deployment/src/test/java/io/quarkiverse/wiremock/devservice/WiremockExtensionServiceLoadingDisabled.java

```diff
@@ -0,0 +1,33 @@
+
+package io.quarkiverse.wiremock.devservice;
+
+import static io.quarkiverse.wiremock.devservice.WireMockConfigKey.PORT;
+import static org.hamcrest.Matchers.is;
+import static org.jboss.resteasy.reactive.RestResponse.StatusCode.OK;
+
+import org.eclipse.microprofile.config.ConfigProvider;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.quarkus.test.QuarkusUnitTest;
+import io.restassured.RestAssured;
+
+public class WiremockExtensionServiceLoadingDisabled {
+
+    private static final String APP_PROPERTIES = "application-extension-loading-disabled.properties";
```

Comment:
This file is not part of your commit. If this is by intention as extension scanning is disabled by default, I would recommend removing the config definition completely.

Reply:
Oops! I just added it